Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
-name = "transform-output"
+name = "rust_test_runner"
 version = "0.1.0"
 authors = ["Peter Goodspeed-Niklaus <peter.r.goodspeedniklaus@gmail.com>"]
-edition = "2018"
+edition = "2021"
```

Comment:
There are always compiler warnings when some code is doing something that was deprecated / will change in a future edition. Since we didn't get any warnings for 2018, it is safe to upgrade.